src/test/bin/winkernel/control.cpp

```diff
@@ -768,6 +768,11 @@ QuicTestCtlEvtIoDeviceControl(
         QuicTestCtlRun(QuicTestClientDisconnect(Params->StopListenerFirst));
         break;
 
+    case IOCTL_QUIC_RUN_STATELESS_RESETKEY:
```

Comment:
Once you make this define a _new_ code, you should move this to the end, and update `QUIC_IOCTL_BUFFER_SIZES` with the new size.